addons/popochiu/engine/interfaces/i_room.gd

```diff
@@ -241,8 +241,26 @@ func room_readied(room: PopochiuRoom) -> void:
 		chr.self_modulate = Color.from_string(chr_dic.self_modulate, Color.WHITE)
 		chr.light_mask = chr_dic.light_mask
 		chr.baseline = chr_dic.baseline
-		chr.walk_to_point = chr_dic.walk_to_point
-		chr.look_at_point = chr_dic.look_at_point
+		
+		# ---- fix #343 by checking the type of the stored value in the chr_dic Dictionary ---------
+		if chr_dic.has("walk_to_point_x"):
+			chr.walk_to_point = Vector2(chr_dic.walk_to_point_x, chr_dic.walk_to_point_y)
+		elif chr_dic.has("walk_to_point"):
+			if chr_dic.walk_to_point is String:
+				var walk_split := (chr_dic.walk_to_point as String).split_floats(",")
```

Comment:
Should a check be done at this point to make sure that walk_to_point has the right format (i.e. "X,Y")? The script will die if walk_to_point contained a word for example, or didn't have a comma.

Reply:
Could be, but this assumes we're dealing with the data as it was being stored in `popochiu_room_data.gd` prior to this fix. Anyway, I can add the validation; it never hurts.